examples/containerized_app.py

```diff
@@ -22,6 +22,6 @@
 
 with sky.Dag() as dag:
     t = sky.Task(run=run_command, setup=setup_cmd)
-    t.set_resources(sky.Resources(sky.AWS(), accelerators='V100'))
+    t.set_resources(infra='aws', accelerators='V100')
```

Comment:
Seems wrong - Task.set_resources isn't changed in the PR?

```suggestion
    t.set_resources(sky.Resources(infra='aws', accelerators='V100'))
```

Reply:
Oops, good catch! Added